### PR TITLE
Adding the ability to automatically Create, Mount, Boot the Discovery ISO on baremetal servers

### PR DESCRIPTION
**Pull request by rlopez133**
This PR gives the ability to create, mount and boot the discovery ISO for baremetal servers automatically via Ansible playbook. I've included all the details in the README.adoc on how it works.

### CODE REVIEW

**Comment by rlopez133** [comment]
@romfreiman

**Comment by tsorya** [comment]
@rlopez133 i don't really understand ansible to say the truth.
lets push it?

**Comment by rlopez133** [comment]
> @rlopez133 i don't really understand ansible to say the truth.
> lets push it?

@tsorya - let me review it again as I know there have been changes to the Makefile and variables since I pushed this up. As for the Ansible part, let's change that :) Let me know if you want to meet and go over the playbook (15 min should do the trick)

**Comment by rlopez133** [comment]
@tsorya not sure why its failing your CI  but its ready from my perspective and tested.

**Comment by iranzo** [comment]
@tsorya Can your review  after the changes to pass lint? Thx

File: ansible-bm-install/collections/ansible_collections/okd/assisted_installer/MANIFEST.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "collection_info": {
+    "authors": ["Roger Lopez <rlopez@redhat.com>"],
+    "dependencies": {},
+    "description": "Assisted Installer to Create, Mount, Boot Discovery ISO",
+    "documentation": "https://github.com/openshift/assisted-test-infra",
+    "homepage": "https://github.com/openshift/assisted-test-infra",
+    "issues": "https://github.com/openshift/assisted-test-infra/issues",
+    "license": ["GPL-2.0-or-later"],
```
[the remark this answers is not in the document]
Comment (rlopez133) [reply ansible-bm-install/collections/ansible_collections/okd/assisted_installer/MANIFEST.json:9]: @mhrivnak there isn't. These files get auto generated and it must of picked GPL 2.0 as default and I didn't realize.

File: ansible-bm-install/collections/ansible_collections/okd/assisted_installer/galaxy.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+### OPTIONAL but strongly recommended
+
+# A short summary description of the collection
+description: Assisted Installer to Create, Mount, Boot Discovery ISO
+
+# Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
+# accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
+license:
+- GPL-2.0-or-later
```
[the remark this answers is not in the document]
Comment (rlopez133) [reply ansible-bm-install/collections/ansible_collections/okd/assisted_installer/galaxy.yml:31]: @mhrivnak there isn't. These files get auto generated and it must of picked GPL 2.0 as default and I didn't realize.

File: ansible-bm-install/collections/ansible_collections/okd/assisted_installer/roles/boot_iso/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
+
+    - hosts: servers
+      roles:
+         - { role: username.rolename, x: 42 }
+
+## License
+
+BSD
```
[the remark this answers is not in the document]
Comment (rlopez133) [reply ansible-bm-install/collections/ansible_collections/okd/assisted_installer/roles/boot_iso/README.md:27]: generic file created by the role. I will remove these as they aren't required.

File: ansible-bm-install/collections/ansible_collections/okd/assisted_installer/galaxy.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (rlopez133) [reply ansible-bm-install/collections/ansible_collections/okd/assisted_installer/galaxy.yml]: fixed.

File: ansible-bm-install/collections/ansible_collections/okd/assisted_installer/roles/boot_iso_existing_os/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
+
+    - hosts: servers
+      roles:
+         - { role: username.rolename, x: 42 }
+
+## License
+
+BSD
```
[the remark this answers is not in the document]
Comment (rlopez133) [reply ansible-bm-install/collections/ansible_collections/okd/assisted_installer/roles/boot_iso_existing_os/README.md]: got rid of all these README.md that I don't need.

**Pull request merged**